libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js

```diff
@@ -56,12 +56,12 @@ describe('Schema Merge Tests', function () {
     // This will install the latest version of botframework-cli if the schema changed and npm is present.
     // This only runs on Windows platforms.
     maybeIt('should generate a new tests.schema file', async function () {
-        this.timeout(100000);
+        this.timeout(150000);
         fs.unlinkSync(testsSchemaPath);
 
         // Merge all schema files.
         const sdkRoot = path.join(__dirname, '..', '..', '..');
-        const mergeCommand = `bf dialog:merge ${sdkRoot}/libraries/**/*.schema ${sdkRoot}/libraries/**/*.uischema !**/testbot.schema !${sdkRoot}/libraries/botbuilder-dialogs-adaptive/tests/schema/*.* -o ${testsSchemaPath}`;
+        let mergeCommand = `bf dialog:merge ${sdkRoot}/libraries/**/*.schema ${sdkRoot}/libraries/**/*.uischema !**/testbot.schema !${sdkRoot}/libraries/botbuilder-dialogs-adaptive/tests/schema/*.* -o ${testsSchemaPath}`;
```

Comment:
One final suggestion - I tend to prefer this style for long commands:
```typescript
const mergeCommand = [
  'npx -p @microsoft/botframework-cli', // invoke with npx to not alter repo dependencies
  'bf dialog:merge', // dialog merge command
  ... // other flags/arguments on separate lines
].join(' ');
```

Reply:
This whole long string ^ should be broken up and placed in the array, and the keyword should be `const` 😄

Reply:
@joshgummersall [Beat ya](https://github.com/microsoft/botbuilder-js/pull/3404/commits/35e39dc9e9a4b6a5061ba5c329b0d39168628946) 👍

Reply:
Top speed achieved!

Reply:
Not sure if I'm missing something in the diff, but my suggestion is to do:
```typescript
const mergeCommand = [
  'npx -p @microsoft/botframework-cli',
  'bf dialog:merge',
  `${sdkRoot}/libraries/**/*.schema`, // include all schema files
  `${sdkRoot}/libraries/**/*.uischema`, // include all uischema files
  '!**/testbot.schema', // ignore testbot schema files
  `!${sdkRoot}/libraries/botbuilder-dialogs-adaptive/tests/schema/*.*`, // ignore teams schema files
  `-o ${testsSchemaPath}` // write output to test schema path
].join(' ');
```

Reply:
*Actually* [resolved](https://github.com/microsoft/botbuilder-js/pull/3404/commits/f11c032be39cdf2eac9390c68654908068eab964#diff-99d1dec39793b38d549b790cc1bb6f2bc962dd39aee498126f98b5421456e616R65)